manifests/q/quick-lint/quick-lint-js/2.4.1/quick-lint.quick-lint-js.locale.en-US.yaml

```diff
@@ -0,0 +1,36 @@
+# To the extent possible under law, Matthew "strager" Glazar
+# has waived all copyright and related or neighboring rights
+# to _quick-lint-js winget manifests_. This work is
+# published from United States.
+# http://creativecommons.org/publicdomain/zero/1.0/
+
+PackageIdentifier: quick-lint.quick-lint-js
+PackageVersion: 2.4.1
+PackageLocale: en-US
+
+PackageName: quick-lint-js
+PackageUrl: https://quick-lint-js.com/
+ShortDescription: 'Find bugs in JavaScript programs.'
+Moniker: quicklintjs
+
+Publisher: quick-lint
+PublisherSupportUrl: https://quick-lint-js.com/contact/
+Copyright: 'Copyright (C) 2020  Matthew "strager" Glazar'
+License: 'GPL-3.0-or-later,...'
```

Comment:
```suggestion
License: 'GPL-3.0-or-later'
```

Reply:
I included all the licenses for the bundled software. Let me know if this is a problem.